backend/graminsta/post/services.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: UTF-8 -*-
+"""services.py"""
+
+from django.contrib.auth import models as auth_models
+from django.shortcuts import get_object_or_404
+from .serializers import UserSerializer
+from .serializers import FollowSerializer
+from .models import FollowRelationship
+
+
+def get_people_user_follows(username):
+    """
+    Returns JSON format users that the given user follows.
+    """
+    user = auth_models.User.objects.get(username=username)
+    ids = FollowRelationship.objects.filter(from_user=user) \
+        .values_list('to_user_id', flat=True)
+    queryset = auth_models.User.objects.filter(id__in=ids)
+    serializer = UserSerializer(queryset, many=True)
+    return serializer
+
+
+def create_follow_relationship(validated_data):
+    """
+    Creates a new follow relationship.
+    """
+    serializer = FollowSerializer(data=validated_data)
+    serializer.is_valid(raise_exception=True)
+    serializer.save()
+    return serializer
```

Comment:
Return the created `FollowRelationShip` instead of the serializer, caller of this method doesn't care the serializer, but it might care about the created instance.